.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,38 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+  "name": "C# (.NET)",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/dotnet:1-8.0-bookworm",
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {},
+    "ghcr.io/devcontainers/features/powershell:1": {},
+    "ghcr.io/dapr/cli/dapr-cli": {
+      "version": "1.12.0"
+    },
+    "ghcr.io/azure/azure-dev/azd": {}
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "ms-dotnettools.vscodeintellicode-csharp",
+        "EditorConfig.EditorConfig"
+      ]
+    }
+  },
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  // "features": {},
+  // Use 'forwardPorts' to make a list of ports inside the container available locally.
+  // "forwardPorts": [5000, 5001],
+  // "portsAttributes": {
+  //		"5001": {
+  //			"protocol": "https"
+  //		}
```

Comment:
[nit] please change tabs to spaces

Reply:
@RussKie I had the same discussion earlier with @danmoseley [here](https://github.com/dotnet/aspire/pull/3520#discussion_r1560509909). And we agreed on doing it the same as the runtime team is doing. Are there any new information that where discussed internally?

Reply:
The rest of the file uses spaces, and this part uses tabs. It just looks inconsistent. That's all.